mcs/class/System.Web/System.Web.Security/FormsAuthenticationModule.cs

```diff
@@ -71,6 +89,7 @@ public void Dispose ()
 
 		public void Init (HttpApplication app)
 		{
+
```

Comment:
tbh, I couldn't care less about code style in this part of the codebase :laughing: